code/modules/antagonists/demon/sins/envy.dm

```diff
@@ -0,0 +1,40 @@
+/obj/effect/proc_holder/spell/targeted/touch/envy
+	name = "Vanity Steal"
+	desc = "Engulfs your arm in a jealous might, allowing you to steal the look of the first human-like struck with it. Note, the form change is not reversable."
+	hand_path = /obj/item/melee/touch_attack/envy
+	school = "evocation"
+	charge_max = 150
+	clothes_req = FALSE
+	invocation = "ETERNAL FLAMES"
+	invocation_type = "whisper"
+	action_icon = 'icons/mob/actions/actions_changeling.dmi'
+	action_icon_state = "transform"
+	action_background_icon_state = "bg_demon"
+
+/obj/item/melee/touch_attack/envy
+	name = "Envious Hand"
+	desc = "A writhing, burning aura of jealousy, ready to be unleashed."
```

Comment:
```suggestion
	desc = "A writhing, the burning aura of jealousy, ready to be unleashed."
```
This would be more of a personal preference in grammar. Feel free to ignore this one.